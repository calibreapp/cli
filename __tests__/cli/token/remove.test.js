const Configstore = require('configstore')
const {
  runCLI,
  setupIntegrationServer,
  teardownIntegrationServer
} = require('../../utils')

const pkg = require('../../../package.json')
const config = new Configstore(pkg.name)

describe('remove token', () => {
  beforeAll(() => setupIntegrationServer())
  afterAll(() => teardownIntegrationServer())

  test('is removed', async () => {
    config.set('token', '123')
    const out = await runCLI({ args: 'token remove', testForError: true })
    expect(out).toMatchSnapshot()
  })

  test('no token to remove', async () => {
    const out = await runCLI({ args: 'token remove', testForError: true })
    expect(out).toMatchSnapshot()
  })
})
