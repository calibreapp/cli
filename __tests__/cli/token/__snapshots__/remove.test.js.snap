// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`remove token is removed 1`] = `""`;

exports[`remove token no token to remove 1`] = `""`;
